doc/whats_new.rst

```diff
@@ -54,6 +54,11 @@ Bug
 - Fix bug where :func:`~hnn_core.network.pick_connection` did not return an
   empty list when searching non existing connections, by `Nick Tolley`_ in :gh:`515`
 
+- Fix bug in :meth:`~hnn_core.NetworkBuilder.aggregate_data` where continuous 
```

Comment:
should we just say that it created intermittent failures with MPIBackend without mention of `NetworkBuilder` object? I think that's mostly for internal use, no?

Reply:
I feel like what I put is more explicit, but I’ll defer to your judgment re: the appropriate scope for whats_new.rst (it’s still a little hazy in my head how anyone but an advanced user might take advantage of the documentation in this file).

Reply:
This document is supposed to be user facing ... e.g., when you install a new release, what are the things you need to be aware of as a user. We sometimes go into the details too much. Another purpose is to give folks credit for their work

Reply:
Okay, what about this: https://github.com/jonescompneurolab/hnn-core/pull/545/commits/3528e69686fc7f17e5bbebf0a79c64c937ce3d20?

Reply:
Maybe still too much detail? :grimacing: